### PR TITLE
Created list contacts UI

### LINKED ISSUE #5: Create list contacts UI

### PR DESCRIPTION
**Pull request by Ishaan-Singhal**
Fixes #5 
Wheel 2022-09-27 11-52-19

@yedhink 
Please review.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/ContactMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+
+import { Search, Plus, Settings } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+import { CONTACT_MENU_BAR_VALUES } from "./constants";
+
+const Menu = ({ showMenu }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/ContactMenu.jsx:9]: - Go through the Slack thread and understand why redundant namespacing is not good.
- Also, the React component name should be exactly same as the file name.

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          render: () => (
+            <Dropdown buttonStyle="text" icon={MenuHorizontal}>
+              <div className="p-1.5">
+                <li>Edit</li>
+                <li>Delete</li>
+              </div>
+            </Dropdown>
+          ),
+        },
+      ]}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table.jsx:61]: Move this out of the presentation layer into a separate, appropriate, file.

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+          dataIndex: "id",
+          key: "id",
+          width: "10%",
+          render: () => (
+            <Dropdown buttonStyle="text" icon={MenuHorizontal}>
+              <div className="p-1.5">
+                <li>Edit</li>
+                <li>Delete</li>
+              </div>
+            </Dropdown>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table.jsx:58]: Go through the Slack thread.

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+          render: monthDateFormatter,
+        },
+        {
+          title: "",
+          dataIndex: "id",
+          key: "id",
+          width: "10%",
+          render: () => (
+            <Dropdown buttonStyle="text" icon={MenuHorizontal}>
+              <div className="p-1.5">
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Table.jsx:54]: Don't add unnecessary tags. What's the purpose of this `div` here?

**Review by yedhink — COMMENTED** [review]
@Ishaan-Singhal

File: app/javascript/src/components/Dashboard/Contacts/ContactMenu.jsx (hunk 1 above)
Comment (Ishaan-Singhal) [reply app/javascript/src/components/Dashboard/Contacts/ContactMenu.jsx:9]: Ok, I will replace the files and the components with proper naming.

File: app/javascript/src/components/Dashboard/Contacts/Table.jsx (hunk 2 above)
Comment (Ishaan-Singhal) [reply app/javascript/src/components/Dashboard/Contacts/Table.jsx:61]: I will go through the steps suggested in slack thread for the same and make appropriate changes.

**Pull request merged**